articles/application-gateway/application-gateway-autoscaling-zone-redundant.md

```diff
@@ -90,6 +92,8 @@ Capacity unit price = 744(hours) * Max (compute unit Max(25/50 for connections/s
 
 Total price = $267.84 + $85.71 = $353.55
 
+Note: The Max function returns the largest value in a pair of values.
```

Comment:
```suggestion
> [!NOTE]
> The Max function returns the largest value in a pair of values.
```

Reply:
@tksh164
Please incorporate this markdown formatting change for a Note and I'll then merge.

Reply:
@vhorne 
Done. I have reformatted another "Note".